harness/build.gradle

```diff
@@ -55,6 +55,12 @@ dependencies {
     testCompile(group: 'org.assertj', name: 'assertj-core', version: project.'assertj-core.version')
 }
 
+compileJava {
+    options.fork = true
+    options.forkOptions.executable = 'javac'
+    options.compilerArgs << '-XDignore.symbol.file'
```

Comment:
This is sort of a deprecated way to ignore warnings. I would just pull it. We have lots of similar warnings in our main sources.

Reply:
i double checked, and without these the build generates an error.  
```error: package sun.security.tools.keytool does not exist 
import sun.security.tools.keytool.CertAndKeyGen;```

without this flag, javac won't include some of the older sun classes